dhcpv4/options.go

```diff
@@ -255,6 +255,10 @@ type OptionHumanizer struct {
 func (oh OptionHumanizer) Stringify(code uint8, data []byte) string {
 	c := oh.CodeHumanizer(code)
 	val := oh.ValueHumanizer(c, data)
+	switch optionCode(code) {
```

Comment:
This is a bit unideal... can you indent this like Vendor data was indented?

Reply:
I don't quite follow what do you mean by indent this like vendor data, this PR indent option82 output like following:
```
    Relay Agent Information:        
        Agent Circuit ID Sub-option: cid-1([99 105 100 45 49]) 
        Agent Remote ID Sub-option: rid-1([114 105 100 45 49])
```
could you give an example how above should be indented?

Reply:
He means that there should not be a switch here and this should come from the option itself.
Maybe with a specific ralay agent information specific `ValueHumanizer`.